pkg/webhook/resources/upgrade/validator.go

```diff
@@ -224,6 +238,37 @@ func (v *upgradeValidator) checkManagedCharts() error {
 	return nil
 }
 
+// Since volume snapshot/backup will hold one additional LH VA tickets, it can block VM live migration.
+// We should check if there is any vmbackup under processing before upgrade
+func (v *upgradeValidator) checkVMBackups() error {
+	vmBackups, err := v.vmBackupCache.GetByIndex(indexeres.VMBackupByIsProgressing, strconv.FormatBool(true))
+	if err != nil {
+		return err
+	}
+
+	if len(vmBackups) == 0 {
+		return nil
+	}
+
+	return werror.NewBadRequest(fmt.Sprintf("vmbackup %s/%s is under processing", vmBackups[0].Namespace, vmBackups[0].Name))
+}
+
+// Since volume snapshot/backup will hold one additional LH VA tickets, it can block VM live migration,
+// and an active schedule could start a vmbackup at any time.
+// we should check if there is any running schedule before upgrade
+func (v *upgradeValidator) checkScheduleVMBackups() error {
+	svmbackups, err := v.svmbackupCache.GetByIndex(indexeres.ScheduleVMBackupBySuspended, strconv.FormatBool(false))
```

Comment:
Is following case already prevented:

When an upgrade is on-going, the suspended status is changed from false to true.

Following function seems not checking.

```
func (v *scheuldeVMBackupValidator) Update(_ *types.Request, oldObj runtime.Object, newObj runtime.Object) error {
```

Reply:
This PR is to prevent the unexpected VM backup from being triggered. Once the PR is merged, only user manually resumes the schedule during the upgrade can make a schedule back to running as an upgrade is on-the-fly.

However, Harvester document already mentions `Do not operate the cluster during an upgrade` https://docs.harvesterhci.io/v1.4/upgrade/index/#start-an-upgrade